core/src/main/scala/dagr/core/config/Configuration.scala

```diff
@@ -104,124 +51,19 @@ private[core] object Configuration extends ConfigurationLike {
   * that is then mixed into the Configuration object above, and used to create a concrete
   * Configuration trait below.
   */
-private[config] trait ConfigurationLike extends LazyLogging {
-  private[config] def config : Config
+private[config] trait ConfigurationLike extends LazyLogging with dagr.sopt.util.ConfigurationLike {
```

Comment:
I think it was a hold over from when the `Configuration` object was private.  I'll clean it up.